recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: scikit-image-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/s/scikit-image/scikit-image-{{ version }}.tar.gz
-  md5: 04ea833383e0b6ad5f65da21292c25e1
+  md5: c92b682ed77b568218423b7afe454996
```

Comment:
We can now get `sha256` from [pypi.org]( https://pypi.org/project/scikit-image/#files ).